cinema/views.py

```diff
@@ -118,12 +140,16 @@ class OrderPagination(PageNumberPagination):
     max_page_size = 100
 
 
-class OrderViewSet(viewsets.ModelViewSet):
+class OrderViewSet(mixins.CreateModelMixin,
+                   mixins.ListModelMixin,
+                   viewsets.GenericViewSet):
     queryset = Order.objects.all().prefetch_related(
         "tickets__movie_session__movie", "tickets__movie_session__cinema_hall"
     )
     serializer_class = OrderSerializer
     pagination_class = OrderPagination
+    authentication_classes = (TokenAuthentication,)
+    permission_classes = [IsAuthenticated]
```

Comment:
```suggestion
    permission_classes = (IsAuthenticated)
```

use the same parentheses

Reply:
After change brackets, tests have failed. I got back

Reply:
add comma after class, so it will work with brackets